common/src/main/java/com/wynntils/features/alwayson/TradeMarketPriceConversionFeature.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.alwayson;
+
+import com.wynntils.core.features.StateManagedFeature;
+import com.wynntils.mc.event.ChatSentEvent;
+import com.wynntils.utils.StringUtils;
+import com.wynntils.wynn.event.ChatMessageReceivedEvent;
+import com.wynntils.wynn.utils.WynnUtils;
+import java.util.regex.Pattern;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+public class TradeMarketPriceConversionFeature extends StateManagedFeature {
```

Comment:
Why state managed? Give the user the option to disable. StateManagedFeature is only used in really specific scenarios, not when it is not logical to disable the feature.